CMakeLists.txt

```diff
@@ -275,10 +321,10 @@ if ( NOT fletch_ENABLED_Boost)
   list( APPEND KWIVER_LIBRARY_DIRS "${BOOST_ROOT_ARG}/lib")
 endif()
 
-if (fletch_ROOT)
+if (fletch_FOUND)
   # since we have fletch, add include and lib dirs to export
-  list( APPEND KWIVER_INCLUDE_DIRS "${fletch_ROOT}/include")
-  list( APPEND KWIVER_LIBRARY_DIRS "${fletch_ROOT}/lib")
+  list( APPEND KWIVER_INCLUDE_DIRS "${fletch_DIR}/include")
+  list( APPEND KWIVER_LIBRARY_DIRS "${fletch_DIR}/lib")
```

Comment:
I believe fletch_ROOT is the correct symbol here since it really points to the fletch install area. fletch_DIR may also point there, but it does not have to. fletch_DIR is used by the find_module script to point to the directory that contains fletchConfig.cmake